action.yml

```diff
@@ -55,14 +59,14 @@ runs:
     - name: Run maven
       run: |-
         test -z "${{ inputs.run-tests }}" && SKIP_TESTS="-DskipTests"
-        mvn -B package ${SKIP_TESTS}
+        mvn -B "${{ inputs.maven-additional-options }}" package ${SKIP_TESTS}
```

Comment:
I have a bad feeling about these quotes: Consider passing multiple options in `maven-additional-options`, wouldn't `mvn` just see them as one argument if you wrap them in double quotes?

Same remark for all other instances of quotes below (except the shell script invocation).

Reply:
Yes you are right: The other options like maven-release-options are not quoted here. I will fix it.